torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1229,8 +1229,12 @@ XLATensorPtr clamp(const XLATensorPtr& input,
       Clamp(input->GetIrValue(), min_max.min, min_max.max));
 }
 
-XLATensorPtr clone(const XLATensorPtr& input) {
-  XLATensorPtr cloned = input->CreateFrom(input->GetIrValue());
+XLATensorPtr clone(const XLATensorPtr& input, bool is_conj) {
```

Comment:
Why does the clone method implementation depend on conj? This seems like a layering violation.

Reply:
Because that's where conjugate is supposed to be materialized. PyTorch's conjugate is implemented by a dispatch-key, making it lazy. It's only reflected to the actual data on `clone()` calls.

Reply:
I see. Could we factor the `ir = torch::lazy::Value(torch_xla::MakeNode<ConjCopy>(ir));` into a separate `conj_copy` function instead of adding a `is_conj` flag to `Clone`? I understand the need to match PyTorch semantics, but it would still be good to avoid mixing concerns.